x-pack/plugins/ingest_manager/server/services/setup.ts

```diff
@@ -80,6 +81,9 @@ async function createSetupSideEffects(
   ) {
     throw new Error('Policy not found');
   }
+
+  await ensureAgentActionPolicyChangeExists(soClient);
```

Comment:
IMO it's a bit strange to have this function called in this order, in the middle of the logic for ensuring default packages for default policy. what do you think of moving it to L68, before the start of this logic, but after the rest of the `ensure()` methods?

Reply:
we probably want to have it after the package installation so the config contains the package I should probably move it L111